README.md

```diff
@@ -1,10 +1,23 @@
 # Varkes
-*Varkes* means small boats in contrast to the big ships. We are building a framework/toolkit of components, which then mock the big ships (applications). The aim is to give a framework to application writers that can mock their application so that they can experiment on Kyma functions without thinking about the deployment of their applications. Once they have the required logic they want from Kyma, they can deploy their applications without any change on Kyma side.
-## App Connector Client
-App connector has a REST api that is used to get certificate from kyma cluster. You can run it with `npm start` or you can use the *Dockerfile* to create a docker image. Further instructions are in [app-connector-client/README.md](app-connector-client/README.md).
 
-## OpenAPI Mock
-Mocks REST calls given an OpenAPI specification. [openapi-mock/README.md](openapi-mock/README.md).
+## Overview
 
-## Odata Mock
-Mocks rest calls given an OData specification [odata-mock/README.md](odata-mock/README.md).
\ No newline at end of file
+The Varkes (Greek for **small boats**) framework mocks the existing applications. The framework allows you to mock the external applications and experiment with Kyma functionality without thinking about their deployment. Use the framework to set up connection between Kyma and the mock application, and deploy the applications without any change on the Kyma side.

```

Comment:
I somehow like the wording of the old intro section, can we keep more of it:

*Varkes* means small boats in contrast to the big ships. We are building a framework/toolkit of components, which then mock the big ships (applications). The aim is to give a framework to application writers that can mock their application so that they can experiment on Kyma functions without thinking about the deployment of their applications. Once they have the required logic they want from Kyma, they can deploy their applications without any change on Kyma side.

First it plays a bit with small and big ships and it expresses the idea of playing around till you have a productive version which then can be used with the big ship. Somehow that two aspects are lost in the new version.

Reply:
I updated the doc in this respect as well :)